### PR TITLE
v7.22.3

### PR DESCRIPTION
**Pull request by alexfauquette**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,71 @@
...
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐞 Bugfixes
+- 🌍 Improve Chinese, Spanish, Swedish, and Turkish locales on the Data Grid
+
+Special thanks go out to the community contributors who have helped make this release possible:
+@CarlosLopezLg, @headironc, @viktormelin, @qerkules, @DungTiger.
+Following are all team members who have contributed to this release:
+@alexfauquette, @LukasTy, @MBilalShafi.
```
Comment (flaviendelangle) [inline CHANGELOG.md:18]:
```suggestion
@alexfauquette, @LukasTy, @MBilalShafi, @flaviendelangle.
```

:angel:

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,71 @@
...
+
+- [charts-pro] Fix missing typeOverload (#15400) @alexfauquette
+
+### Docs
+
+- [docs] Add `PickersPopper` component to customization playground (#15397) @LukasTy
+- [docs] Add `next` version links (#15423) @LukasTy
+
+- [code-infra] Tentative fix for Argos flaky screenshot tests (#15399) @JCQuintas
+- [docs-infra] Transpile .ts demo files (#15421) @KenanYusuf
```
Comment (flaviendelangle) [inline CHANGELOG.md:69]:
```suggestion
- [docs-infra] Transpile `.ts` demo files (#15421) @KenanYusuf
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,71 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.22.3
+
+_Nov 21, 2024_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
```
Comment (alexfauquette) [inline CHANGELOG.md:10]:
```suggestion
We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
```

**Review by flaviendelangle — APPROVED** [review]
:ok_hand: Nothing else to note

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,71 @@
...
+
+#### `@mui/x-date-pickers-pro@7.22.3` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-date-pickers@7.22.3`, plus:
+
+- [DateTimeRangePicker] Use time in `referenceDate` when selecting date (#15431) @LukasTy
+
+### Charts
+
+#### `@mui/x-charts-pro@7.0.0-beta.8` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
```
Comment (LukasTy) [inline CHANGELOG.md:59]: What's the plan for the stable release?
Shouldn't we do it given that we have started the work on the next major? 🤔

**Review by LukasTy — COMMENTED** [review]
Should we wait for https://github.com/mui/mui-x/pull/15535 @cherniavskii? 🤔

File: CHANGELOG.md (hunk 4 above)
Comment (alexfauquette) [reply CHANGELOG.md:59]: It's not mandatory, but effectively the package is quite stable now that we work on the next version

**Comment by cherniavskii** [comment]
@alexfauquette Could you sync the release branch with v7.x? We have merged a few cherry picks recently.

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,72 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.22.3
+
+_Nov 21, 2024_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 📊 Charts Pro get stable. The zoom and Heatmap are now stable.
```
Comment (cherniavskii) [inline CHANGELOG.md:12]:
```suggestion
- 📊 Charts Pro get stable. The [zoom](https://mui.com/x/react-charts/zoom-and-pan/) and [Heatmap](https://mui.com/x/react-charts/heatmap/) are now stable.
```

**Review by cherniavskii — APPROVED** [review]

File: packages/x-charts-pro/package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-charts-pro",
-  "version": "7.0.0-beta.7",
+  "version": "7.22.3",
```
Comment (LukasTy) [inline packages/x-charts-pro/package.json:3]: Would it make sense to release the base charts package to have the versions in sync and the version selector in the header represent correct data? 🤔

**Review by KenanYusuf — APPROVED** [review]

**Review by LukasTy — APPROVED** [review]

**Pull request merged**